extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -209,41 +212,33 @@ export class InstrumentationService {
    * @param {!AnalyticsEventListenerDef} The callback to call when the event
    *   occurs.
    * @param {!JSONType} config Configuration for instrumentation.
+   * @param {boolean} visibilityState State for which the callback is triggered.
    * @private
    */
-  createVisibilityListener_(callback, config) {
+  createVisibilityListener_(callback, config, visibilityState) {
     if (config['visibilitySpec']) {
       if (!isVisibilitySpecValid(config)) {
         return;
       }
 
-      this.runOrSchedule_(() => {
-        visibilityFor(this.win_).then(visibility => {
-          visibility.listenOnce(config['visibilitySpec'], vars => {
-            callback(new AnalyticsEvent(AnalyticsEventType.VISIBLE, vars));
-          });
-        });
+      visibilityFor(this.win_).then(visibility => {
+        visibility.listenOnce(config['visibilitySpec'], vars => {
+          callback(new AnalyticsEvent(AnalyticsEventType.VISIBLE, vars));
```

Comment:
good catch. The event type is not used anywhere in the code. but it should have been different.